### DISCUSSION TITLE
/oper command not work

### DISCUSSION THREAD

**Discussion by sunung0110** [inspircd/inspircd#1936] [Support]
**Description**

**Steps to reproduce the issue:**

1.use hexchat login method set custom
2.custom command /oper %u %p(username and password)
3.not work

**Describe the results you received:**

Access denied by configuration

**Describe the results you expected:**

login to operator

**Additional information you deem important (e.g. issue happens only occasionally):**

if use password hash, not work(both of oper and user)
and normal password, work(user)/not work(oper)
i use sha256 hash

**Output of `./bin/inspircd --version`:**

InspIRCd-3.11.9-deb1

**Comment by sadiepowell** [comment]
>Access denied by configuration

This is a connect class error. Are you trying to log in to the server using a server operator account? You can't do that.
